### PR TITLE
Tutorial Compile VGMPlay under Android

### PR DESCRIPTION
**Pull request by GuckTubeYT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,3 +63,11 @@ make
 3. Done.
 
 Thanks to grauw for macOS compilation instructions.
+
+## Compile VGMPlay under Android
+1. Install [Termux](https://en.wikipedia.org/wiki/VGM_(file_format)) on Play Store
```
Comment (ValleyBell) [inline README.md:68]: The link destination is wrong.

Comment (GuckTubeYT) [reply README.md:68]: Oh im sorry

**Comment by GuckTubeYT** [comment]
Ok fixed, thanks for remind me

**Comment by GuckTubeYT** [comment]
Btw, if you find bad grammad in README.md, can you fix it?

**Pull request merged**